src/GridLines.tsx

```diff
@@ -222,3 +221,91 @@ const WeekStripes = ({ monthStart, timeScale }: WeekStripesProps) => {
 
   return <g>{lines}</g>
 }
+
+/* ·················································································································· */
+/*  Hour
+/* ·················································································································· */
+
+interface HourLineProps {
+  xPosition: number
+  height?: string
+}
+
+const HourLine = ({ xPosition, height }: HourLineProps) => {
+  const xAxisTheme = useTimelineTheme().xAxis
+  const classes = useMonthViewStyles(xAxisTheme)
+  return (
+    <line
+      className={classes.line}
+      x1={xPosition}
+      y1={0}
+      x2={xPosition}
+      y2={height ? height : '80%'}
+      strokeWidth={1}
+    />
+  )
+}
+
+const TEN_SECOND_OFFSET_MS = 10000;
+interface HourViewProps {
+  height: number
+  domain: [number, number]
+  timeScale: ScaleLinear<number, number>
+}
+
+const defaultHourViewLabelFontSize = 10
+
+const useHourViewStyles = makeStyles((theme: Theme) => ({
+  ...gridLineStyle(theme),
+  label: (xAxisTheme: XAxisTheme) => ({
+    fill: xAxisTheme.labelColor,
+    opacity: 0.5,
+    fontFamily: theme.typography.caption.fontFamily,
+    fontSize: xAxisTheme.hourLabelFontSize ? xAxisTheme.hourLabelFontSize : defaultHourViewLabelFontSize,
+    fontWeight: xAxisTheme.hourLabelFontWeight ? xAxisTheme.hourLabelFontWeight : 'bold',
+    textAnchor: 'middle',
+    cursor: 'default',
+  }),
+}))
+
+const getTimelineBoundsLabel = (date: Date) => {
+  const time = date.toLocaleTimeString();
+  // +1 because months start at 0
+  const month = date.getMonth() + 1;
+  const day = date.getDate();
+  const label = `${month}/${day} @ ${time}`;
+  return label;
+}
+
+const HourView = ({ height, domain, timeScale }: HourViewProps) => {
+  const xAxisTheme = useTimelineTheme().xAxis
+  const classes = useHourViewStyles(xAxisTheme)
+
+  // Scale the bounds slightly inside so they don't touch the edges
+  const leftBoundMs = domain[0] + TEN_SECOND_OFFSET_MS;
+  const rightBoundMs = domain[1] - TEN_SECOND_OFFSET_MS;
+
+  const leftBoundLabel = getTimelineBoundsLabel(new Date(leftBoundMs));
+  const rightBoundLabel = getTimelineBoundsLabel(new Date(rightBoundMs));
+
+  const leftBoundPos = timeScale(leftBoundMs)!
+  const rightBoundPos = timeScale(rightBoundMs)!
+
+  // TODO: Possibly change key
```

Comment:
I tried various key values however having it like this seemed to work (other things I tried were like the seconds or milliseconds, which just causes rendering problems because it can't find the key after rendering once)

Reply:
I think it's fine. You could also just use text for these (e.g. `start` and `end`)